pkg/routeagent_driver/constants/constants.go

```diff
@@ -50,6 +50,7 @@ const (
 	// the egress traffic to the corresponding CNIInterfaceIP on that host.
 	RouteAgentHostNetworkTableID = 150
 
-	NATTable    = "nat"
-	FilterTable = "filter"
+	NATTable         = "nat"
+	FilterTable      = "filter"
+	ForceMssClamping = "submariner.io/mssClamping"
```

Comment:
IMO its better to declare this constant at the same place where we are declaring other constants - i.e., https://github.com/submariner-io/submariner/blob/8e8728427df87ef146fa1243008dc6e72f078b04/pkg/apis/submariner.io/v1/types.go#L92

Also, what is your opinion on the following name?
submariner.io/tcp-mss-clamping